presentation/src/main/java/com/iluwatar/presentation/App.java

```diff
@@ -0,0 +1,26 @@
+package com.iluwatar.presentation;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * a class used to start this demo.
```

Comment:
Here it would be great to explain the pattern briefly and how this example implements it